CalibrationPlannerTestA:
  acceleration: 0.1
  forward_only: true
  goal_timeout: 20.0
  kd_ang: 0.5
  kd_lat: 1.23
  kd_lon: 0.0
  ki_ang: 0.0
  ki_ang_max: 62.0
  ki_lat: 0.06153
  ki_lat_max: 62.0
  ki_lon: 0.0
  ki_lon_max: 10.0
  kp_ang: 6.1538
  kp_lat: 49.2307
  kp_lon: 10.0
  max_cmd_vel_ang: 3.1
  max_cmd_vel_speed: 1.0
  max_follow_distance: 1.0
  max_goal_angle_error: 20.0
  max_goal_distance_error: 0.2
  restore_defaults: false
  speed_angular: 45.0
  speed_fast: 0.4
  speed_fast_threshold: 1.0
  speed_fast_threshold_angle: 20.0
  speed_slow: 0.15

CalibrationPlannerTestB:
  acceleration: 1.0
  forward_only: false
  goal_timeout: 20.0
  kd_ang: 0.0
  kd_lat: 0.0
  kd_lon: 0.0
  ki_ang: 0.0
  ki_ang_max: 10.0
  ki_lat: 0.0
  ki_lat_max: 10.0
  ki_lon: 0.0
  ki_lon_max: 10.0
  kp_ang: 1
  kp_lat: 6.15
  kp_lon: 6.0
  max_cmd_vel_ang: 3.1
  max_cmd_vel_speed: 0.6
  max_follow_distance: 1.0
  max_goal_angle_error: 20.0
  max_goal_distance_error: 0.1
  restore_defaults: false
  speed_angular: 40.0
  speed_fast: 0.1
  speed_fast_threshold: 5.0
  speed_fast_threshold_angle: 5.0
  speed_slow: 0.1

CalibrationPlanner:
  acceleration: 0.1
  forward_only: true
  goal_timeout: 20.0
  kd_ang: 0.1
  kd_lat: 0.1
  kd_lon: 0.1
  ki_ang: 0.0
  ki_ang_max: 0.0
  ki_lat: 0.0
  ki_lat_max: 0.0
  ki_lon: 0.0
  ki_lon_max: 0.0
  kp_ang: 10
  kp_lat: 20
  kp_lon: 3 # 3
  max_cmd_vel_ang: 1.0 
  max_cmd_vel_speed: 0.2
  max_follow_distance: 1.0
  max_goal_angle_error: 10.0
  max_goal_distance_error: 0.2
  restore_defaults: false
  speed_angular: 25.0
  speed_fast: 0.1
  speed_fast_threshold: 0.3
  speed_fast_threshold_angle: 5.0
  speed_slow: 0.1

CalibrationFastPlanner:
  acceleration: 0.1
  forward_only: true
  goal_timeout: 20.0
  kd_ang: 0.1
  kd_lat: 0.1
  kd_lon: 0.1
  ki_ang: 0.0
  ki_ang_max: 0.0
  ki_lat: 0.0
  ki_lat_max: 0.0
  ki_lon: 0.0
  ki_lon_max: 0.0
  kp_ang: 10
  kp_lat: 20
  kp_lon: 10
  max_cmd_vel_ang: 1.0
  max_cmd_vel_speed: 0.2
  max_follow_distance: 1.0
  max_goal_angle_error: 25.0
  max_goal_distance_error: 0.5
  restore_defaults: false
  speed_angular: 25.0
  speed_fast: 0.40
  speed_fast_threshold: 0.3
  speed_fast_threshold_angle: 15.0
  speed_slow: 0.2
